src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,52 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static final String DELIMITER = ",|:";
+
+    public static int splitAndSum(String text) {
+        if (isEmpty(text))
+            return 0;
```

Comment:
```suggestion
        if (isEmpty(text)) {
            return 0;
        }
```

자바 컨벤션에서는 단일 if 문도 미연에 생길 사고를 방지하기 위해
중괄호를 사용하는 걸 추천하는데 적용해보면 어떨까요?